editor/src/components/inspector/sections/layout-section/layout-system-subsection/layout-system-controls.tsx

```diff
@@ -273,34 +288,123 @@ export const PaddingControl = React.memo(() => {
     )
   }, [])
 
+  useSelectorWithCallback(
+    Substores.selectedViews,
+    selectedViewsSelector,
+    () => setOveriddenMode(null),
+    'aa aaaa',
```

Comment:
aaaaaaaaaaaaa! 

it'd be nice if we could kill these two useSelectorWithCallbacks

Reply:
(I fixed the aaaa thing in https://github.com/concrete-utopia/utopia/pull/3289/commits/a312fc16db8d71c0ce30dc25e18e4737262827a8)